### DISCUSSION TITLE
WAN Public IP

### DISCUSSION THREAD

**Discussion by JHarrison712** [netbox-community/netbox#12027] [Q&A]
Good morning everyone,

We are implementing NetBox at our organization and we have multiple public IP in different range say 12.13.14.15,16,19 so we skipped 17 & 18 because we didn't order those at first. Is there an way the Circuit tie into the IPAM for WAN network range? Also can someone please explain VRF?

Thank you,
John

**Comment by candlerb** [comment]
> Is there an way the Circuit tie into the IPAM for WAN network range?

Not directly as far as I know.  If a circuit connects to an interface via a circuit termination, then the interface's IP address will be one of the addresses in the prefix.  But otherwise you can use the comments field, or create a custom field (e.g. a custom [object or multi-object field](https://docs.netbox.dev/en/stable/customization/custom-fields/#custom-object-fields) could point to a Prefix, or a Range, or individual IPAddress objects).

> Also can someone please explain VRF?

VRFs are separate layer 3 routing domains with their own forwarding tables - a bit like how VLANs are isolated, when you're talking about layer 2 networks.

From Netbox's point of view: if you have two customers both using 192.168.1.1, and you put them in different VRFs, then Netbox won't complain that they are conflicting IP addresses.
